crates/integration-tests/src/tests/snapshots/integration_tests__tests__basic__runs_a_query.snap

```diff
@@ -5,43 +5,53 @@ expression: "query(r#\"\n                query Movies {\n                  movie
 data:
   movies:
     - imdb:
-        rating: 6.2
+        rating:
+          $numberDouble: "6.2"
         votes: 1189
       title: Blacksmith Scene
     - imdb:
-        rating: 7.4
+        rating:
+          $numberDouble: "7.4"
         votes: 9847
       title: The Great Train Robbery
     - imdb:
-        rating: 7.1
+        rating:
+          $numberDouble: "7.1"
```

Comment:
The strict Extended JSON serialization for `rating` is expected. In the sample database ratings are a mixture of `double` and `int` values. Because we have two different types in the same position type unification resolves the type to `ExtendedJSON`